docs/troubleshooting.rst

```diff
@@ -13,20 +13,34 @@ communication issue:
 - Ensure that ``ddtrace`` is configured with the hostname and port of the
   agent. See :ref:`Configuration` for the configuration variables.
 
-To verify that a connection is being made to the agent, enable start-up logs
-with the environment variable ``DD_TRACE_STARTUP_LOGS=true``. A diagnostic log
-that looks like ``DATADOG TRACER DIAGNOSTIC...`` will be displayed with the
-connection error if there is a problem connecting to the agent.
+To verify that a connection is being made to the agent, run the command ``ddtrace-run --info`` while your application is running.
```

Comment:
> while your application is running.

this makes it seem like `ddtrace-run --info` reports information about a running application, which it doesn't